dashboard/views.py

```diff
@@ -563,76 +593,81 @@ def get_user_assignment_submission(current_user,assignments_in_course_df, course
 
 
 # don't show the avg scores for student when individual assignment is not graded as canvas currently don't show it
-def no_show_avg_score_for_ungraded_assignments(row):
+def no_show_avg_score_for_ungraded_assignments(row: Dict) -> Union[str, float]:
     if row['score'] is None:
         return 'Not available'
-    else: return row['avg_score']
+    else:
+        return row['avg_score']
 
 
-def user_percent(row):
+def user_percent(row: Dict) -> int:
     if row['graded']:
         s = round((row['score'] / row['points_possible']) * row['towards_final_grade'], 2)
         return s
     else:
         return row['towards_final_grade']
 
 
-def percent_calculation(consider_weight,total_points,hidden_assignments,row):
+def percent_calculation(consider_weight: bool,
+                        total_points: bool,
+                        hidden_assignments: bool,
+                        row: Dict) -> int:
```

Comment:
I think these type hints need to be checked (or removed). I'm pretty sure based on my other work that `consider_weight` is an `int` or (`numpy.int64`). `total_points` appears to be some kind of number. `hidden_assignments` appears to be correct. `row` is probably a `Series` but I have not verified.

Reply:
Okay, when I logged these in the function, I get this:

```
consider_weight
<class 'numpy.int64'>
total_points
<class 'numpy.float64'>
hidden_assignments
<class 'bool'>
row
<class 'pandas.core.series.Series'>
```